libs/deserializer/src/test/java/de/cyface/deserializer/UnzippedPhoneDataDeserializerTest.java

```diff
@@ -66,13 +66,13 @@ void test_unzipped() throws InvalidLifecycleEvents, URISyntaxException, NoSuchMe
 
         // Assert
         assertThat(result, hasProperty("metaData", hasProperty("identifier",
-                is(equalTo(new MeasurementIdentifier("65a6f5dc-459d-4634-a759-197e224f76cf", mid))))));
+                is(equalTo(new MeasurementIdentifier("2aec8af8-08a9-40e7-a86c-e490d33f48d9", mid))))));
```

Comment:
Had to capture a new (small) sample measurement for this with the new `serialzer` format on my Android phone for this.